.github/workflows/release_win.yml

```diff
@@ -106,14 +106,13 @@ jobs:
         cd onnx
         pytest
 
-    - name: Test the uploaded backend test data
+    - name: Test generated backend test data
       run: |
         cd onnx
-        # onnx.checker all existing backend data
-        python workflow_scripts\test_generated_backend.py
         # onnx.checker all generated backend data
         python onnx\backend\test\cmd_tools.py generate-data
-        python workflow_scripts\test_generated_backend.py
+        # Only test generated backend node test data
+        pytest onnx\test\test_backend_test.py -k OnnxBackendNodeModelTest
```

Comment:
what is "-k OnnxBackendNodeModelTest" for?

Reply:
Also earlier test script was running both checker and shape_inference checks on models and now we are only doing the shape_inference check?

Reply:
Specify OnnxBackendNodeModelTest here will make onnx\test\test_backend_test only test those models under `onnx/backend/test/case/node/`.

> Also earlier test script was running both checker and shape_inference checks on models and now we are only doing the shape_inference check?

Yes. To keep the same behavior as previous verification, I just added onnx.checker in test_backend_test.py. It can also enhance the test for backend data.

Reply:
Where is "OnnxBackendNodeModelTest " defined? The name suggests it will run tests for both Node and Model

Reply:
I cannot find the source code about it shortly, but from my understanding the name here is collected as "OnnxBackend-[directory name under [here](https://github.com/onnx/onnx/tree/master/onnx/backend/test/data)]-ModelTest".
For instance, OnnxBackend"Node"ModelTest or OnnxBackend"PyTorchConverted"ModelTest.